bindings/python/parsers/sample-models.hpp

```diff
@@ -71,10 +85,15 @@ namespace se3
 
     inline void SampleModelsPythonVisitor::expose()
     {
-      bp::def("buildSampleModelHumanoidSimple",
-              static_cast <Model (*) (bool)> (&SampleModelsPythonVisitor::buildSampleModelHumanoidSimple),
+      bp::def("buildSampleModelHumanoidRandom",
+              static_cast <Model (*) ()> (&SampleModelsPythonVisitor::buildSampleModelHumanoidRandom),
+              "Generate a (hard-coded) model of a humanoid robot with 6D limbs and random joint placement.\nOnly meant for unit tests."
+              );
+
+      bp::def("buildSampleModelHumanoidRandom",
+              static_cast <Model (*) (bool)> (&SampleModelsPythonVisitor::buildSampleModelHumanoidRandom),
               bp::args("bool (usingFreeFlyer)"),
-              "Generate a (hard-coded) model of a simple humanoid robot."
+              "Generate a (hard-coded) model of a humanoid robot with 6D limbs and random joint placement.\nOnly meant for unit tests."
```

Comment:
"6 DoF" and "random joint placements".

Reply:
This comment was copied from the original C++ code, I'll modify that too

Reply:
Perfect!